src/panels/lovelace/editor/config-elements/hui-generic-entity-row-editor.ts

```diff
@@ -0,0 +1,150 @@
+import "@polymer/paper-input/paper-input";
+import {
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { assert } from "superstruct";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { stateIcon } from "../../../../common/entity/state_icon";
+import "../../../../components/ha-formfield";
+import "../../../../components/ha-icon-input";
+import "../../../../components/ha-switch";
+import { HomeAssistant } from "../../../../types";
+import { EntitiesCardEntityConfig } from "../../cards/types";
+import "../../components/hui-action-editor";
+import "../../components/hui-entity-editor";
+import "../../components/hui-theme-select-editor";
+import { LovelaceRowEditor } from "../../types";
+import {
+  EditorTarget,
+  entitiesConfigStruct,
+  EntitiesEditorEvent,
+} from "../types";
+import { configElementStyle } from "./config-elements-style";
+
+const SecondaryInfoValues = [
+  "",
+  "entity-id",
+  "last-changed",
+  "last-triggered",
+  "position",
+  "tilt-position",
+  "brightness",
+];
+
+@customElement("hui-generic-entity-row-editor")
+export class HuiGenericEntityRowEditor extends LitElement
+  implements LovelaceRowEditor {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @internalProperty() private _config?: EntitiesCardEntityConfig;
+
+  public setConfig(config: EntitiesCardEntityConfig): void {
+    assert(config, entitiesConfigStruct);
+    this._config = config;
+  }
+
+  get _entity(): string {
+    return this._config!.entity || "";
+  }
+
+  get _name(): string {
+    return this._config!.name || "";
+  }
+
+  get _icon(): string {
+    return this._config!.icon || "";
+  }
+
+  get _secondary_info(): string {
+    return this._config!.secondary_info || "";
+  }
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this._config) {
+      return html``;
+    }
+
+    return html`
+      ${configElementStyle}
```

Comment:
This should in `static get styles()`

Reply:
Well
That would mean I need to change it in every config element editor file... Its a Template result

Reply:
We should start somewhere 🤷‍♂️  you can prevent the change everywhere else right now by doing:
```ts
const styles = css`blabla {}`;
const configElementStyle= html`<style>${styles.cssText}</style>`
```